src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -2,13 +2,18 @@
 
 import java.math.BigDecimal;
 import java.util.List;
+import mate.academy.Main;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(Main.class);
```

Comment:
The logger is currently set up with `Main.class`, which is not appropriate for this class. It should be `OrderServiceImpl.class` to correctly reflect the class where the logger is used.